packages/client/hooks/useSortSubColumns.ts

```diff
@@ -0,0 +1,48 @@
+import {useLayoutEffect, useEffect} from 'react'
+import {commitLocalUpdate} from 'react-relay'
+import {SubColumn} from '~/types/constEnums'
+import {GroupingKanbanColumn_reflectionGroups} from '~/__generated__/GroupingKanbanColumn_reflectionGroups.graphql'
+import useAtmosphere from './useAtmosphere'
+
+const useSortSubColumns = (
+  isWidthExpanded: boolean,
+  reflectionGroups: GroupingKanbanColumn_reflectionGroups
+) => {
+  const atmosphere = useAtmosphere()
+  useLayoutEffect(() => {
+    if (!reflectionGroups) return
+    commitLocalUpdate(atmosphere, (store) => {
+      reflectionGroups.forEach((group, index) => {
+        const reflectionGroup = store.get(group.id)
+        if (!reflectionGroup) return
+        const subColumn = index % 2 === 0 ? SubColumn.LEFT : SubColumn.RIGHT
```

Comment:
what if the retro is a single column? can we allow them to expand the column to be 3+ columns?